packages/devtools_app/lib/src/memory/memory_heap_tree_view.dart

```diff
@@ -227,9 +227,9 @@ class HeapTreeViewState extends State<HeapTree> with AutoDisposeMixin {
       ]);
     } else if (controller.snapshotByLibraryData != null) {
       if (controller.showHeatMap.value) {
-        snapshotDisplay = HeatMapSizeAnalyzer(
+        snapshotDisplay = TreemapSizeAnalyzer(
           child: SizedBox.expand(
```

Comment:
is this SizedBox.expand still necessary? Can we just do `snapshotDisplay = MemoryTreemap(controller);`?